### PR TITLE
Updated user guide to include steps from doc

### PR DESCRIPTION
**Pull request by rodsan**
as promised during the call today, I have converted the user guide from the word doc as markdown and am now submitting a PR (along with a few grammatical changes). Once this is up, we can further contribute, clean up further and make it even better.

### CODE REVIEW

**Comment by rodsan** [comment]
made some additional commits to add the guide to the readme file... no major changes were made, just minor aesthetic + grammatical changes

File: README.md [hunk 1]
```diff
@@ -1,63 +1,40 @@
...
+3. Execute output in `bld\bin\BinSkim.Driver` for testing
 
-Submitting Pull Requests
-========================
-Run `test.cmd` at the root of the enlistment to ensure that all tests pass, and x64 and x86 release builds succeed.
+### Submit Pull Requests
 
-Getting Started as a User
-=========================
+1. Run `test.cmd` at the root of the enlistment to ensure that all tests pass, and x64 and x86 release builds succeed
```
Comment (michaelcfanning) [inline README.md]: >test.cmd [](start = 8, length = 8)

Actually called BuildAndTest.cmd

File: docs/UserGuide.md [hunk 2]
```diff
@@ -1,16 +1,148 @@
...
+
+#### -p, --plug-in
+
+The `-p` or `--plug-in` argument is used to provide a path to a BinSkim plug-in that will be loaded and invoked at analysis time, in addition to the built-in checks. This argument can be specified multiple times on the command-line.
+
+## BinSkim Release History
+
+The latest version is always available on the main at **[NuGet](https://www.nuget.org/packages/Microsoft.CodeAnalysis.BinSkim/)**
+
+| Version | Details |
```
Comment (michaelcfanning) [inline docs/UserGuide.md]: > Version | Detai [](start = 1, length = 16)

We should drop the version info which is not up to date. 

We could refer to ReleaseHistory.md instead, which has this info (this information probably better suited to readme.md)

File: README.md (hunk 1 above)
Comment (rodsan) [inline README.md]: changed!

File: README.md [hunk 3]
```diff
@@ -1,63 +1,40 @@
-BinSkim Binary Analyzer
-=======================
+# BinSkim Binary Analyzer
```
Comment (michaelcfanning) [inline README.md]: >BinSkim Binary Analyzer [](start = 2, length = 23)

Did this file actually really get moved to a folder named README.MD?

**Comment by michaelcfanning** [comment]
LGTM. Feel free to merge.

---
In reply to: [324090181](https://github.com/Microsoft/binskim/pull/97#issuecomment-324090181) [](ancestors = 324090181)

File: docs/UserGuide.md (hunk 2 above)
Comment (rodsan) [inline docs/UserGuide.md]: will do!

File: README.md (hunk 3 above)
Comment (rodsan) [inline README.md]: this file was there from the beginning. I just touched it up a bit. The User guide was the one I worked on a lot

**Comment by rodsan** [comment]
as promised, made the 2 changes requested: changed the cmd tool name and added link to the release md file.

**Comment by rodsan** [comment]
I don't have merging right on Binskim, so you would have to do it @michaelcfanning. It's grayed out for me.

**Pull request merged**